src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
+$button-color: #fff;
+$button-bl-color: #00acdc;
+$button-text-gr-color: #616070;
+$button-big-text: #060b35;
+$button-widht: 166px;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 32px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $button-big-text;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    margin-bottom: 4px;
+    align-items: flex-start;
+    width: $button-widht;
+  }
+
+  &__product {
+    display: flex;
+    align-items: center;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    margin-bottom: 16px;
+    color: $button-text-gr-color;
+    width: $button-widht;
+  }
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $button-big-text;
+  margin-bottom: 24px;
+  width: $button-widht;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  filter: grayscale(100%);
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: $button-widht;
+  height: 18px;
+  margin-bottom: 16px;
+
+  &__title {
+    font-weight: 400;
+    font-size: 12px;
+    color: $button-text-gr-color;
+  }
+  &__money {
+    font-weight: 700;
+    font-size: 16px;
+  }
+}
+
+.button {
```

Comment:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/c742de45-de33-49a6-850c-6d2d47f2a372">
you should have only two blocks and each in it own file, pls rewrite it

Reply:
<img width="191" alt="Screenshot 2024-08-08 at 17 51 33" src="https://github.com/user-attachments/assets/decd5250-b856-46f8-a332-229b5f6ebe14">

like this one? Only with card and price?

Reply:
yes, exactly
mentor meant that you need to have 2 scss files: `stars` and `card`
where you out all css

Reply:
@lerastarynets ok, and all the other styles I need to leave in index.scss? like in index.scss add @import to card.scss and stars.scss before the styles? or create an additional folder in the blockc folder and add the stars.scss, card.scss folder and the style.scss folder where all the other styles will be? 
![image](https://github.com/user-attachments/assets/6b6b5bd7-bc1e-4767-973c-a1245be27c79)